examples/plot_simulate_somato.py

```diff
@@ -57,35 +64,63 @@
 # estimate the location and amplitude of a single current dipole. At the
 # moment, we do not offer explicit recommendations on which source
 # reconstruction technique is best for HNN. However, we do want our users
-# to note that the dipole currents simulate with HNN are assumed to be normal
-# to the cortical surface. Hence, using the option ``pick_ori='normal'``
-# seems to make most sense.
-
-method = "MNE"
+# to note that the dipole currents simulated with HNN are assumed to be normal
+# to the cortical surface. Hence, using the option ``pick_ori='normal'`` is
+# appropriate.
 snr = 3.
 lambda2 = 1. / snr ** 2
-stc = apply_inverse(evoked, inv, lambda2, method=method, pick_ori="normal",
-                    return_residual=False, verbose=True)
+evoked = epochs.average()
+stc = apply_inverse(evoked, inv, lambda2, method='MNE',
+                    pick_ori="normal", return_residual=False,
+                    verbose=True)
 
-pick_vertex = np.argmax(np.linalg.norm(stc.data, axis=1))
+###############################################################################
+# To extract the primary response in primary somatosensory cortex (S1), we
+# create a label for the postcentral gyrus (S1) in source-space
+hemi = 'rh'
+label_tag = 'G_postcentral'
+label_s1 = mne.read_labels_from_annot(subject, parc='aparc.a2009s', hemi=hemi,
+                                      regexp=label_tag,
+                                      subjects_dir=subjects_dir)[0]
+
+###############################################################################
+# Visualizing the distributed S1 activation in reference to the geometric
+# structure of the cortex (i.e., plotted on a structural MRI) can help us
+# figure out how to orient the dipole. Note that in the HNN framework,
+# positive and negative deflections of a current dipole source correspond to
+# upwards (from deep to superficial) and downwards (from superficial to deep)
+# current flow, respectively.
+stc_label = stc.in_label(label_s1)
+brain = stc_label.plot(subjects_dir=subjects_dir, hemi='rh', surface='white',
+                       smoothing_steps='nearest', view_layout='horizontal',
+                       initial_time=0.04, backend='pyvista')
+brain.add_label(label_s1, borders=True)
+
+###############################################################################
+# Now we extract the pca-flipped time course from S1. Note that the most
+# prominent component of the median nerve response occurs in the posterior wall
+# of the central sulcus at ~0.040 sec. Since the dipolar activity here is
```

Comment:
The "most prominent component" is a field pattern corresponding to an anterior-posterior dipole, right? And if it's on the posterior wall, the current direction then is deep-to-superficial, i.e., 'negative' in HNN parlance? Did I get that right?

It's quite a lot of detail to gloss over in half a sentence... If you can, feel free to use a few more words. But don't get bogged down with trying to over-simplify a complex matter.

Reply:
> The "most prominent component" is a field pattern 

I hesitate to use the term 'field pattern' because I don't want people to get EM fields mixed up with current dipoles (even though they are intimately tied), the latter of which is explicitly modeled with an inverse solution.

> corresponding to an anterior-posterior dipole, right? And if it's on the posterior wall, the current direction then is deep-to-superficial, i.e., 'negative' in HNN parlance? Did I get that right?

Kind of. You're right that this is anterior-to-posterior (i.e., backwards pointing) but this corresponds to *superficial-to-deep* current flow in the posterior wall of the CS.

> It's quite a lot of detail to gloss over in half a sentence... If you can, feel free to use a few more words. But don't get bogged down with trying to over-simplify a complex matter.

I'll see what I can do. I'm trying hard not to write a book on it here in this example but some more explanation might be good :joy:

Reply:
> I hesitate to use the term 'field pattern' because I don't want people to get EM fields mixed up with current dipoles (even though they are intimately tied), the latter of which is explicitly modeled with an inverse solution.

I agree we shouldn't mention field patterns here since we don't show them! This was just to remind myself of what we're talking about at

<img width="386" alt="Screenshot 2021-01-30 at 17 04 08" src="https://user-images.githubusercontent.com/5610076/106361253-35a2df80-631d-11eb-82ea-e5eefb0c0bb7.png">


> Kind of. You're right that this is anterior-to-posterior (i.e., backwards pointing) but this corresponds to _superficial-to-deep_ current flow in the posterior wall of the CS.

Ah, yes, you're right: posterior bank, so if dipole is AP, then current direction is sup-to-deep, so negative.